pallets/loans/src/tests/repay_loan.rs

```diff
@@ -900,3 +902,79 @@ fn with_external_pricing() {
 		assert_eq!(current_price(), NOTIONAL);
 	});
 }
+
+#[test]
+fn with_more_than_required_external() {
```

Comment:
Super NIT, but can we move this to line 262 to be closed to `with_more_than_required()` that works similar but with internal pricing?